src/qml/AboutPage.qml

```diff
@@ -16,37 +17,78 @@
  */
 
 import QtQuick 2.9
+import org.asteroid.utils 1.0
 import org.asteroid.controls 1.0
 import org.nemomobile.systemsettings 1.0
 
-Item {
+Flickable {
     AboutSettings {
         id: about
     }
-
-    Icon {
-        name: "logo-asteroidos"
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.topMargin: Dims.h(10)
-        anchors.top: parent.top
-        anchors.bottom: osLabel.top
-        anchors.bottomMargin: Dims.h(3)
-        width: height
-    }
-    Label {
-        id: osLabel
-        text: about.operatingSystemName
-        font.bold: true
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.bottom: releaseLabel.top
+    DiskUsage {
+        id: diskUsage
     }
-    Label {
-        id: releaseLabel
-        text: about.softwareVersion
-        opacity: 0.8
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.bottom: parent.bottom
-        anchors.bottomMargin: Dims.h(7)
+
+    contentHeight: contentcolumn.implicitHeight
+    Column {
+        id: contentcolumn
+        anchors.fill: parent
+        Item { //this acts as a spacer to put the logo in the middle of the screen.
+            height: parent.width*0.1
+            width: height
+        }
+        Icon {
+            name: "logo-asteroidos"
+            anchors.horizontalCenter: parent.horizontalCenter
+            width: parent.width*0.4
+            height: width
+        }
+        Label {
+            id: osLabel
+            text: about.operatingSystemName
+            anchors.horizontalCenter: parent.horizontalCenter
+        }
+        Label {
+            id: releaseLabel
+            text: about.softwareVersion
+            opacity: 0.8
+            anchors.horizontalCenter: parent.horizontalCenter
+        }
+        Repeater {
+            model: [
+                { label: qsTr("Build ID"), text: DeviceInfo.buildID },
+                { label: qsTr("Codename"), text: DeviceInfo.machineName },
+                { label: qsTr("Host name"), text: DeviceInfo.hostname },
+		{ label: qsTr("WLAN MAC"), text: about.wlanMacAddress },
+		{ label: qsTr("IMEI"), text: about.imei },
+                { label: qsTr("Serial number"), text: about.serial },
+		{ label: qsTr("Total disk space"), text: qsTr("%L1").arg(Math.round(about.totalDiskSpace() / 1e7)/100) + " GB" },
+		{ label: qsTr("Available disk space"), text: qsTr("%L1").arg(Math.round(about.availableDiskSpace() / 1e7)/100) 
+                            + " GB (" + (100.0 * about.availableDiskSpace() / about.totalDiskSpace()).toFixed(0) + "%)" },
+		{ label: qsTr("Display size"), text: Dims.w(100) + qsTr("W") + " x " + Dims.h(100) + qsTr("H") },
+		{ label: qsTr("Kernel version"), text: kernelversion },
+		{ label: qsTr("Qt version"), text: qtversion }
```

Comment:
You fixed the setContextProperty but not the usages it seems ?

Reply:
Fixed, thanks!  I realize that my test failed to clear the QML cache.